src/Container/ChirperMessage.cs

```diff
@@ -0,0 +1,48 @@
+﻿using ICities;
+using UnityEngine;
+
+namespace CSM.Container
+{
+    /// <summary>
+    /// Implementation of the IChirper Interface.
+    /// This allows displaying messages to the user using the Chirper at the top of the screen.
+    /// </summary>
+    public class ChirperMessage : IChirperMessage
+    {
+        uint IChirperMessage.senderID { get => _senderID; }
+        string IChirperMessage.senderName { get => _senderName; }
+        string IChirperMessage.text { get => _text; }
```

Comment:
```suggestion
        public uint IChirperMessage.senderID { get => _senderID; }
        public string IChirperMessage.senderName { get => _senderName; }
        public string IChirperMessage.text { get => _text; }
```
Right? Or can that be excluded in this case?

Reply:
Good question. It does work fine without declaring them `public`.
While the `IChirper` interface itself is declared public, the properties aren't (at least not explicitely). That probably is why.